### ISSUE TITLE
standardize unit tests in conch.test.test_recvline.Arrows

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5256]
|[<img alt="djfroofy's avatar" src="https://avatars.githubusercontent.com/u/89760?s=50" width="50" height="50">](https://github.com/djfroofy)| @djfroofy reported|
|-|-|
|Trac ID|trac#5256|
|Type|enhancement|
|Created|2011-09-07 03:06:46Z|

This is a small patch to change tests methods in `test_recvline.Arrows` to abide by naming convention `test_camelCase" and also add docstring to each.

Attachments:

* [stdize-tests-5256.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/4ac/4ac7f0107ada07797a925b4d27ea601359cddf1d/7d80f438d63b0cb288cc94d9190f89c807b111cc.patch) (5037 bytes) - added by djfroofy on 2011-09-07 03:08:04Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5256 5256
type__enhancement enhancement
reporter__djfroofy djfroofy
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1315364806000000 1315364806000000
changetime__1315785978000000 1315785978000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

Looks great, thanks for taking the time.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [32524]) Apply stdize-tests-5256.patch to bring recvline tests up to modern coding standard.

Author: djfroofy
Reviewer: exarkun
Fixes: [#5256](https://github.com/twisted/twisted/issues/5256)

Update the test method names for tests covering `twisted.conch.recvline`,
as well as giving them docstrings describing the functionality they cover.
